libgadget/allvars.h

```diff
@@ -176,6 +144,11 @@ extern struct global_data_all_processes
     /* some SPH parameters */
 
     int DesNumNgb;		/*!< Desired number of SPH neighbours */
+    /*Look like black hole parameters but really for neighbour finding*/
```

Comment:
Blackhole parameters used in neighbour finding? Also could you add a space between * and the text?

Reply:
Yes - neighbour finding for black holes is different for SPH particles. These parameters belong in the density module.